packages/forms-demo/src/components/Menu/index.js

```diff
@@ -27,7 +27,10 @@ export default connect(
         </div>
         <div className={css(styles.settingsContainer)}>
           {Object.keys(settings).map((settingKey, index) => {
-            return <SettingsCheckbox key={index} path={`app.settings.${settingKey}`} />
+            if (settings[settingKey] === Object(settings[settingKey])) {
+              return <SettingsCheckbox key={index} path={`app.settings.${settingKey}`} />
```

Comment:
@edgesoft  Except "Disable submit when form is invalid" I think, the other settings option should be a radio button, because the user must select only one choice. What do you think?